django_lti_tool_provider/tests/test_views.py

```diff
@@ -61,13 +62,15 @@ def get_incorrect_lti_payload(self, path='/lti/', method='POST', data=None):
         req['oauth_signature'] += '_broken'
         return req.to_postdata()
 
-    def send_lti_request(self, payload):
-        return self.client.post('/lti/', payload, content_type='application/x-www-form-urlencoded')
+    def send_lti_request(self, payload, client=None):
+        if not client:
+            client = self.client
```

Comment:
@Kelketek I believe more idiomatic way to do that is

    client = client if client else self.client